### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Plzlvm3aloeSpower** [community/community#59579] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Someone used github and copilot to basically mirror my phone, and they stole a bunch of money from me through fraud because I was paid through ,U.S. bank. I want them out of my phones and accounts plz help they took everything I had

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Plzlvm3aloeSpower 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
